src/__tests__/loadable-components/intersection-observer.test.js

```diff
@@ -40,17 +45,63 @@ describe('Loadable', () => {
     expect(loadable.loadableReturn).toHaveBeenCalledWith(props)
   })
 
+  test('it clears out tracked elements when they become visible', () => {
+    const Loader = loadableVisiblity(opts)
+
+    const wrapper = mount(<Loader {...props} />)
+
+    expect(trackedElements.length).toEqual(1)
+
+    makeElementsVisible()
+
+    expect(trackedElements.length).toEqual(0)
+  })
+
   test('preload calls loadable load', () => {
     loadableVisiblity(opts).load()
 
     expect(loadable().load).toHaveBeenCalled()
   })
 
+  test('preload will cause the loadable component do be displayed', () => {
```

Comment:
Thanks for adding this test! Can you add it for the `react-loadable` section as well?

Also minor typo: should be `to` instead of `do`